source/includes/harvest/_job_posts.md

```diff
@@ -143,6 +144,8 @@ Describes the online job posts for an organization's jobs (as seen on the Job Bo
 |-----------|-------------|
 | id | Job post ID
 | active | If `false`, this job post has been deleted.
+| live | If `true`, this job post status is set to live.
```

Comment:
Hmm. Is this supposed to be here?

Reply:
I added it because it wasn't clear to me what the difference between active and live was

Reply:
Ah - because we never documented it. Excellent, thanks.